horde/model_reference.py

```diff
@@ -169,6 +182,26 @@ def has_nsfw_models(self, model_names):
         #     return True
         return False
 
+    @property
+    def _compvis_ref_url(self):
+        ref_json = DEFAULT_HORDE_IMAGE_COMPVIS_REFERENCE
+        if datetime.now(timezone.utc) <= datetime(2024, 9, 30, tzinfo=timezone.utc):
+            # Flux Beta
+            # I don't understand how this hack works, but perhaps HORDE_IMAGE_COMPVIS_REFERENCE is unset in prod

```

Comment:
Yes, it's unset in prod. This env var is to allow private hordes to use their own references. You can delete this comment